src/service/survey-service.ts

```diff
@@ -0,0 +1,125 @@
+import { t } from "i18next";
+import { LunaticData } from "interface/lunatic/Lunatic";
+import { lunaticDatabase } from "service/lunatic-database";
+import { getCurrentPageSource } from "service/orchestrator-service";
+
+const datas = new Map<string, LunaticData>();
+const activitySurveysIds = ["activitySurvey1", "activitySurvey2", "activitySurvey3"];
+const workingTimeSurveysIds = ["workingSurvey1", "workingSurvey2"];
+const surveysIds = [...activitySurveysIds, ...workingTimeSurveysIds];
+
+const enum FieldNameEnum {
+    lastName = "LASTNAME",
```

Comment:
Use uppercase ?
https://google.github.io/styleguide/jsguide.html#features-objects-enums
(the same for L13/L14)